images/users/triip.svg

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Thanks for your PR :+1:

Could you pass your svg run through [svgo](https://jakearchibald.github.io/svgomg/) to optimize it?